dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -171,9 +171,10 @@ def test_diff_primitive_compilers(self):
                 "stddev": 3.285085839971525,
                 "t-test": {
                     "t-statistic": 0.4155260166386663,
-                    "conservative": {"df": 1, "p-value": 0.749287157907667},
+                    "conservative": {"df": 1.0, "p-value": 0.749287157907667},
```

Comment:
I'm not sure this should be a float.

Reply:
I just checked L561 in `numerical_column_stats.py` and it shows the below... So, I think we are good with `1.0`?

```
results["conservative"]["df"] = float(conservative_df)
```

Reply:
Hmm nvm then!